src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs

```diff
@@ -83,6 +83,11 @@ public sealed class MutabilityInspector {
 			MutabilityInspectionFlags flags,
 			HashSet<ITypeSymbol> typeStack
 		) {
+			if( type is IErrorTypeSymbol  || type == null ) {
+				// type can't be resolved; the project is either not configured correctly, or there's a typo
+				return false;

```

Comment:
I'm conflicted. On one hand, it's conservative, so we should make it unsafe. On the other hand, a seemingly safe type `ImmutableArray<LangTerm>` could be marked unsafe, because some type within `LangTerm` can't be resolved because the project isn't configured correctly. 

In the end I chose to not confuse the dev. There is a third option, which is `throw`, which will show the dev an `analyzer threw exception` message; the issue with that is the analyzer could run mid-word typing (e.g., briefly pause after writing `Immutable`), which would hit this case.